unit_tests/test_grid_fee_calculation.py

```diff
@@ -144,9 +144,37 @@ def test_grid_fee_is_calculated_correctly_for_device_to_parent(self, fee_type):
                                       2.2: ['Load 1.2', 'House 1.2', ]}
         else:
             leaf_name_expected_fee = {2.1: ['House 2.1', 'Load 2.1'],
-                                      1.2: ['Load 1.2', 'House 1.2', ]}
+                                      1.2: ['Load 1.2', 'House 1.2']}
         for expected_fee, leaf_names in leaf_name_expected_fee.items():
             assert isclose(expected_fee, self.grid_fee_calc.calculate_grid_fee(
                 start_market_or_device_name=leaf_names[0],
                 target_market_or_device_name=leaf_names[1],
                 fee_type=fee_type))
+
+    @parameterized.expand([['last_market_fee'], ['next_market_fee']])
```

Comment:
Why not directly this?
```suggestion
    @parameterized.expand(['last_market_fee', 'next_market_fee'])
```

Reply:
Let me check that